pymc3/tests/test_step.py

```diff
@@ -483,7 +487,7 @@ def teardown_class(self):
     @pytest.mark.xfail(condition=(theano.config.floatX == "float32"), reason="Fails on float32")
     def test_sample_exact(self):
         for step_method in self.master_samples:
-            self.check_trace(step_method)
+                self.check_trace(step_method)
```

Comment:
too much intentation
(maybe `black` the file)

Reply:
Fixed the indentation and blackened all newly added code